internal/operands/data-sources/reconcile_test.go

```diff
@@ -157,6 +157,41 @@ var _ = Describe("Data-Sources operand", func() {
 				ExpectResourceNotExists(&cron, request)
 			})
 
+			It("should create DataImportCron in other namespace", func() {
+				cronTemplate.Namespace = "other-namespace"
+				request.Instance.Spec.CommonTemplates.DataImportCronTemplates = []ssp.DataImportCronTemplate{cronTemplate}
+
+				_, err := operand.Reconcile(&request)
+				Expect(err).ToNot(HaveOccurred())
+
+				createdDataImportCron := cdiv1beta1.DataImportCron{}
+				err = request.Client.Get(request.Context, client.ObjectKey{
+					Name:      cronTemplate.GetName(),
+					Namespace: cronTemplate.GetNamespace(),
+				}, &createdDataImportCron)
+				Expect(err).ToNot(HaveOccurred())
+				Expect(createdDataImportCron.Spec).To(Equal(cronTemplate.Spec))
+			})
+
+			It("should remove DataImportCron if template removed from SSP CR in other namespace", func() {
+				cronTemplate.Namespace = "other-namespace"
+				request.Instance.Spec.CommonTemplates.DataImportCronTemplates = []ssp.DataImportCronTemplate{cronTemplate}
+
+				_, err := operand.Reconcile(&request)
+				Expect(err).ToNot(HaveOccurred())
+
+				cron := cronTemplate.AsDataImportCron()
+				cron.Namespace = "other-namespace"
```

Comment:
I think this line is not needed, the namespace should be filled by `AsDataImportCron()` method.